lib/bitreserve/api/contact.rb

```diff
@@ -0,0 +1,34 @@
+module Bitreserve
+  module API
+    module Contact
+      def all_contacts
+        request_data = RequestData.new(
+          Endpoints::USER_CONTACTS,
+          Entities::Contact,
+          authorization_header
+        )
+        Request.perform_with_objects(:get, request_data)
+      end
+
+      def find_contact(id: nil)
+        request_data = RequestData.new(
+          Endpoints::USER_CONTACTS + "/#{id}",
+          Entities::Contact,
+          authorization_header
+        )
+        Request.perform_with_object(:get, request_data)
+      end
+
+      def create_contact(**args)
```

Comment:
I was trying to think of a way to avoid duplication. With that solution we'd have the list of possible arguments both here and in `Entities::Contact`, which might also be a problem if they eventually change. Thoughts?